### PR TITLE
Responsive Mobile and Desktop version.

### PR DESCRIPTION
**Pull request by Ademola-coding**
I have completed the responsive Mobile and Desktop versions for the project.

please, don't forget this is part of our instruction.

- **IMPORTANT NOTE** some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). **Please ignore this part of the design template and use only one input for the full name for all versions**.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -233,7 +359,52 @@ <h2 class="conta">CONTACT US</h2>
...
+               <input type="email" id="mail-D" name="user_email" placeholder="  Email Address" required>
+             
+             
+               <textarea cols="33" rows="16"  id="msg-D" name="user_message" 
+               placeholder=" 
+     Write your message here" maxlength="500" required ></textarea>
+            
+             <div class="button">
+               <button type="submit" id="bttn-D"> send </button>
+             </div>
```
Comment (blessedjasonmwanza) [inline index.html:385]: - [ ] **Required:** Great work on the form it's almost done 👏🏽 . Just minor color changes - When a user focus on the input fields, the color should change and matches the design. here are some pictures for your reference:

## Expected Look
image

## Your current design (Looks like you're using black 👀 )

image

File: index.html [hunk 2]
```diff
@@ -182,13 +288,41 @@ <h3>Multi-Post Stories</h3>
...
+     <li class="skill">css</li>
+     <li class="vector">.</li>
+     <li class="skill">html</li>
+     <li class="vector">.</li>
+     <li class="skill">bootstrap</li>
+     <li class="vector">.</li>
+     <li class="skill">ruby</li>
+  </ul>
+  
+     <button class="btn" type="button"> See Project </button>
```
Comment (blessedjasonmwanza) [inline index.html:315]: - [ ] This is awesome.. Just a little change 😃  - The color for the `See project text` should also match the one specified in your design. The goal is for us to come up with apps that look exactly like the designed [mockups](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A976).

File: CSSfiles/Mstyle.css [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+  .title-work-D {
+    display: none;
+  }
+
+  .line {
+    position: relative;
+    background-color: rgb(39, 42, 42);
+    height: 1px;
+    width: 90%;
+    color: rgb(3, 3, 104);
```
Comment (blessedjasonmwanza) [inline CSSfiles/Mstyle.css:196]: - [ ] **Required:** Oops 🙈 ! Looks like the color of the splitter/line breakers is a little different from the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A976). Kindly check the color codes [using the inspect feature](https://help.figma.com/hc/en-us/articles/360055203533-Use-the-Inspect-panel), to help you match the exact look.

**Review by blessedjasonmwanza — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Ademola-coding 
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @blessedjasonmwanza in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by muneebulrehman — APPROVED** [review]
Hi @Ademola-coding , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good PR description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**